scripts.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line  no-unused-vars
+function openMenu() {
+  document.getElementById('ani').style.marginLeft = '100%';
+  window.location.href = 'menu.html';
+}
+
+// eslint-disable-next-line  no-unused-vars
+function headding() {
+  window.location.href = 'index.html';
+}
+
+// eslint-disable-next-line  no-unused-vars
+function worksSection() {
+  window.location.href = 'index.html#works';
+}
+
```

Comment:
You did a good job working on this milestone! 🚀 I know you've put some effort into this mobile menu, but there are still some things that need to be changed in order to continue with the next milestones.

- [ ] One of the main goals of this project is that you are able to manipulate DOM elements based on events fired by the user (e.g. a click), but the approach you chose here doesn't interact with the document but with the window instead, causing a reload every time you open or close the menu which is something that you wouldn't want to happen regarding the user experience. Please consider fixing this! 😱 

💡 Here are my suggestions:

-  Instead of using two separate HTML files, one for your page and the other for the menu, you can use just one for your page and include the menu there hidden (you could use the CSS `display: none` property to achieve this.
- Instead of changing the `window.location` every time the user opens and closes the menu, you could just add a class, for example `"active"` that overrides the `display: none` for `display: block` when the user clicks on the menu icon, and remove it when the user clicks on the close icon. This way, you assign the responsibility for showing or hiding the menu to your CSS, ending up with a more performant code without reloads, improving tremendously the user experience.


You could refer to this simple [tutorial](https://www.youtube.com/watch?v=flItyHiDm7E).


Keep up the good work! 🤓 🛠️